inference-engine/src/vpu/graph_transformer/src/middleend/special_stage_processor.cpp

```diff
@@ -318,6 +318,25 @@ void SpecialStageProcessor::processReshape(
         input = inputCopy;
     }
 
+    auto needAddCopyBeforeOutput = output->usage() == DataUsage::Output;
```

Comment:
What is the nature of this fix? Is this idea that we don't support outputs with strides? If so, it looks natural to me that we want to introduce `Copy` stage here, but at the same time the issue is not specific to `Reshape` stage - so I expect it'd be fixed in a separate pass that inserts copy by finding output data object with strides without constraints on output stage type.

Could you please explain?

Reply:
So, nature of this fix is the we can't have `Reshape` stage with output with data type `Output`. AFAIK it is `Reshape` specific and more it is case specific(only for Reshapes which not handled on device side - so-called special stage), so no need to create separate pass to solve it.

Reply:
Thanks for clarification!

Reply:
Is this possible two copy stages will be introduced?

Reply:
What do you mean?

Reply:
Seems I understood your point. No, I think it is impossible. Because we can have only one reshape with output data type equal `Output`

Reply:
I mean, I see two calls of `addCopyStage` in this function. Is it possible both of them would be in place?

Reply:
Good point, I'll try it.

Reply:
Neighbour code has been changed and I stayed my changes in previous form to better readability.

Reply:
I don't see changes in code around. Could you please help me find that?

Reply:
Sorry, seems I wrong. But I still going to stay my changes, because it more readable. My part and part bellow have many difference parts and difference logic.

Reply:
Yea, but still there is a question: is it possible both copy stages would be inserted?

Reply:
We discuss it with @AndrewBakalinIntel and has been solve to rework this place and investigate it more intensive.